doc/Manuals/ug-cs_x-road_6_central_server_user_guide.md

```diff
@@ -413,6 +417,63 @@ To change the Central Server address, follow these steps.
   - In case of federated X-Road systems, download the external configuration source anchor and distribute the anchor along with the anchor’s hash value to the federation partners.
   - Reconfigure the management services addresses in the management service Security Server.
 
+## 4.4 Managing the TLS certificates
+
+Access rights: Security Officer
+
+### 4.4.1 Registration and Management Service TLS certificate
+
+Registration and Management Service TLS certificate is used
+- to secure the communication of management service provider (Security Server) and Central Server management services;
+- to the authentication certificate registration request is submitted from the Security Server.
+
+To see Registration and Management Service TLS certificate info, follow these steps.
+
+1. In the Navigation tabs, select Settings --> TLS certificates.
+2. Locate the Management service TLS certificate section and click certificate hash.
+
+To download Management Service TLS certificate, follow these steps.
+
+1. In the Navigation tabs, select Settings --> TLS certificates.
+2. Locate the Management service TLS certificate section.
+3. Click button Download certificate and save the prompted file.
+
+To re-create Management Service key and self-signed certificate, follow these steps.
+
+1. In the Navigation tabs, select Settings --> TLS certificates.
+2. Locate the Management service TLS certificate section and click button Re-create key.
+3. Confirm the re-creating by clicking Confirm.
+4. Do after activities in section [4.4.1.1 Necessary activities after changing certificate](#4411-necessary-activities-after-changing-certificate)
+
+To generate Management Service certificate signing request, follow these steps.
+
+1. In the Navigation tabs, select Settings --> TLS certificates.
+2. Locate the Management service TLS certificate section and click button Generate CSR.
+3. Read the information and enter Distinguished name and click button Generate CSR.
+4. Save prompted file into a safe place.
+
+To upload Management Service certificate, follow these steps.
+
+1. In the Navigation tabs, select Settings --> TLS certificates.
+2. Locate the Management service TLS certificate section and click button Upload certificate.
+3. Find the proper certificate file and click Open, to finish certificate uploading click button Upload.
+4. Do after activities in section [4.4.1.1 Necessary activities after changing certificate](#4411-necessary-activities-after-changing-certificate)
+
+#### 4.4.1.1 Necessary activities after changing certificate
+
+1. When the key and certificate are rotated, and mTLS is enabled between the management Security Server and the management services, the new certificate must be updated to the management Security Server.
+
+2. Reload the nginx service for the certificate change to take effect.
+
+```bash
+systemctl reload nginx
```

Comment:
Could we automate reloading Nginx when a new key is generated and when a certificate is uploaded? In that way, there wouldn't be need to do it manually. Alternatively, changes in the key and certificate files could be checked regularly (e.g., every 5 minutes) and Nginx reloaded automatically if the files have changed. Something like that was implemented for global conf TLS certificates in this PR:

https://github.com/nordic-institute/X-Road/pull/1817/files